### ISSUE TITLE
Address user feedback about user options, etc

### ISSUE THREAD

**Issue by johngallo** [linkagescape/linkage-mapper#86]
there is some great feedback here inthe wiki that needs to be made into several issues for development: https://github.com/linkagescape/linkage-mapper/wiki/Ideas-and-Feedback-for-Development

**Comment by venkanna37** [comment]
John, I can not open the above link. This link is readable by everyone? or made any changes?

**Comment by dkav** [comment]
Wiki is disabled. I have a copy of the feedback if it is needed.

**Comment by dkav** [comment]
John's Wiki entry below -

Feedback from a Respected Colleague June 21, 2019
--
Note: this feedback was written for me, not the public, but I want to get this out here before it gets buried. Initial feedback is in italics.
 
_The inability to re-run LP once it's been run on a project folder was frustrating. Fortunately, my first runs were on copies of my original folders, so I didn't have to go through the process of rerunning LM (which is an HOURS-long process with my datasets). However, it would be nice to be able to re-use a project folder for an LP run. At the very least, I think there should maybe be a user warning about this because I found that once a run is done (or even fails after being mostly done) any other run I attempted on that project folder would fail immediately._
First step here is to check to see if this happens in Arc Cat AND Arc Map, or just Arc Map
 
_Having to make copies of the project folder for each LP run was very cumbersome from both a time and storage space perspective. Again, due to large file/folder sizes, it would take at least 20 minutes to make a copy, then anywhere between 30 min to 4 hours for LP to run. Rinse, repeat. A few times I had to stop and just make copies of my folders for an hour and a half to be able to get through a few runs in a row. It would be great to streamline this if a user could just make a renamed copy of the LCP file and name the LP output and have it write to the same project directory._
 
_Being able to write more than one LP file to a project folder would also make it easier to quickly compare results. Instead, I found I was digging through several file paths to pull up multiple outputs to compare and decide on my inputs and weights for final runs._
 
_It would be great to have the corridor priority value return as a default output in addition to blended priority. I thought this is what I would get as an output based on the user guide, but it looks like that disappeared in one of the newer releases. I wanted to see what the straight priority value was and ALSO to see it combined with my linkages rather than having to weight the linkage priority as 1 and blended priority as 0 to get that output. That now requires multiple runs with multiple folders, etc., which is onerous based on the issues I raised above._
Users should have the option of running the model for a few minutes longer to produce the above output
 
_When LP is geoprocessing and you have a large project directory, Spatial Analyst writes a massive amount of files to your ArcGIS install directory. On several of my computers that are set up with SSDs, which runs my applications, this quickly maxed out space and caused the run to fail. At the very least, I think users should be warned about this to be able to set up a scratch workspace, but ideally, I think that might be something worth writing into the script so the user selects that when inputting values for the LP run._
Needs to be investigated and clarified

**Comment by dkav** [comment]
Further info from @johngallo:

> The first three comments can probably all be assigned to a single issue, and all three get addressed within it.  WE might already have such an issue started.
> 
> The fourth comment is an excellent comment, and I've come across that need my self several times.  The vision if for the user to be able to check a box saying they want that additional input.  IT will take more time to get it (as it is many commands) but will be worth it in most cases. 
> 
> The fifth comment can be addressed a tthe same time that comments 1-3 are being addressed, not sure if it shuodl be folded into that issue or seperate.  Granularity of issues is always a grey zone and subjet to programmer instinct.
> 
> Note: she was not using the most recent version, but I am pretty sure we have not touched any of those issues in the subsequent 2 years.

**Comment by johngallo** [comment]
Comment #4, above, was made into its own issue: https://github.com/linkagescape/linkage-mapper/issues/150
